packages/api/src/models/animalBatchSexDetailModel.js

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import baseModel from './baseModel.js';
+
+class AnimalBatchSexDetailModel extends baseModel {
+  static get tableName() {
+    return 'animal_batch_sex_detail';
+  }
+
+  static get idColumn() {
+    return 'id';
+  }
+
+  // Optional JSON schema. This is not the database schema! Nothing is generated
+  // based on this. This is only used for validation. Whenever a model instance
+  // is created it is checked against this schema. http://json-schema.org/.
+  static get jsonSchema() {
+    return {
+      type: 'object',
+      required: ['batch_id', 'count', 'sex_id'],
+      properties: {
+        batch_id: { type: 'integer' },
```

Comment:
Here I'd maybe go for `animal_batch_id` since the table is called `animal_batch` (and it's not this same table but a different one so not implicit)

Reply:
ok so this follows the db architecture laid out so I have updated the table column name and the model.